src/commands/frameworks-stacks-delete.ts

```diff
@@ -0,0 +1,42 @@
+import { Command } from "../command";
+import { Options } from "../options";
+import { needProjectId } from "../projectUtils";
+import { FirebaseError } from "../error";
+import * as gcp from "../gcp/frameworks";
+import { promptOnce } from "../prompt";
+import * as utils from "../utils";
+
+export const command = new Command("stacks:delete")
```

Comment:
q: why are these still called `stack`?

both in command names, in variable names, etc. Should it be backend now?